StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1001UnitTests.cs

```diff
@@ -164,6 +163,32 @@ public class SA1001UnitTests : CodeFixVerifier
         }
 
         [Fact]
+        public async Task TestSpaceBeforeCommaWhenPartOfInterpolationAlignmentClauseAsync()
+        {
+            string statement = @"var x = new[] { 1, 2, 3, 4, 5, 6, 7 };
+            var t = $""{x[2] ,3}"";";
+            string fixedStatement = @"var x = new[] { 1, 2, 3, 4, 5, 6, 7 };
+            var t = $""{x[2],3}"";";
+
+            DiagnosticResult expected = this.CSharpDiagnostic().WithArguments(" not", "preceded").WithLocation(8, 29);
+
+            await this.TestCommaInStatementOrDeclAsync(statement, expected, fixedStatement).ConfigureAwait(false);
+        }
+
+        [Fact]
+        public async Task TestSpaceAfterCommaWhenPartOfInterpolationAlignmentClauseAsync()
+        {
+            string statement = @"var x = new[] { 1, 2, 3, 4, 5, 6, 7 };
+            var t = $""{x[2], 3}"";";
+            string fixedStatement = @"var x = new[] { 1, 2, 3, 4, 5, 6, 7 };
+            var t = $""{x[2],3}"";";
+
+            DiagnosticResult expected = this.CSharpDiagnostic().WithArguments(" not", "followed").WithLocation(8, 28);
+
+            await this.TestCommaInStatementOrDeclAsync(statement, expected, fixedStatement).ConfigureAwait(false);
+        }
```

Comment:
Please add a testcase for "{x[2] , 3}" as well to verify that a double fault is detected properly as well.

Reply:
➡️ Covered in `TestSpaceBeforeAndAfterCommaWhenPartOfInterpolationAlignmentClauseAsync`